src/controllers/console.js

```
@@ -451,44 +450,64 @@ class ConsoleController extends BaseController {
         };
     });
 
-    handleGetSMSCost = this.handleRESTAsync(async (req) => {
-        const { date } = req.query;
-        if (!date) {
-            throw new APIError(Constants.RESPONSE_ERROR_BAD_REQUEST, "date is required");
+    handleListOwnerApp = this.handleRESTAsync(async (req) => {
+        const { startDate, endDate } = req.query;
+        if (isNaN(Date.parse(startDate)) || isNaN(Date.parse(endDate))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)